### PR TITLE
Add missing relationships to device resource

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

File: api-reference/beta/resources/device.md [hunk 1]
```diff
@@ -81,11 +81,13 @@ This resource lets you add your own data to custom properties using [extensions]
...
+|memberOf|[directoryObject](directoryobject.md) collection|Groups that this device is a member of. HTTP Methods: GET (supported for all groups). Read-only. Nullable.|
 |registeredOwners|[directoryObject](directoryobject.md) collection| The user that cloud joined the device or registered their personal device. The registered owner is set at the time of registration. Currently, there can be only one owner. Read-only. Nullable.|
 |registeredUsers|[directoryObject](directoryobject.md) collection| Collection of registered users of the device. For cloud joined devices and registered personal devices, registered users are set to the same value as registered owners at the time of registration. Read-only. Nullable.|
-|commands | [command](command.md) collection | Set of commands sent to this device|
-|usageRight|[usageRight](usageright.md) collection|Represents the usage rights a device has been granted. |
+|[transitiveMemberOf](../api/device-list-transitivememberof.md) |[directoryObject](directoryobject.md) collection| List the groups that the device is a member of. This operation is transitive. |
```
Comment (Lauragra) [inline api-reference/beta/resources/device.md:89]: This looks like an operation but it's in the Relationships section, which should list nav properties. Double check?

File: api-reference/v1.0/resources/device.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ This resource lets you add your own data to custom properties using [extensions]
 |:---------------|:--------|:----------|
 |extensions|[extension](extension.md) collection|The collection of open extensions defined for the device. Read-only. Nullable.|
 |memberOf|[directoryObject](directoryobject.md) collection|Groups that this device is a member of. HTTP Methods: GET (supported for all groups). Read-only. Nullable.|
-|[List transitive memberOf](../api/device-list-transitivememberof.md) |[directoryObject](directoryobject.md) collection| List the groups that the device is a member of. This operation is transitive. |
+|[transitiveMemberOf](../api/device-list-transitivememberof.md) |[directoryObject](directoryobject.md) collection| List the groups that the device is a member of. This operation is transitive. |
```
Comment (Lauragra) [inline api-reference/v1.0/resources/device.md:68]: This is a nav property; shouldn't link to an operation topic and probably needs a different description.

Comment (FaithOmbongi) [reply api-reference/v1.0/resources/device.md:68]: Updated to _List device transitiveMemberOf_. Hope this is OK?

File: api-reference/beta/resources/device.md [hunk 3]
```diff
@@ -81,11 +81,13 @@ This resource lets you add your own data to custom properties using [extensions]
...
 |extensions|[extension](extension.md) collection|The collection of open extensions defined for the device. Read-only. Nullable.|
+|memberOf|[directoryObject](directoryobject.md) collection|Groups that this device is a member of. HTTP Methods: GET (supported for all groups). Read-only. Nullable.|
 |registeredOwners|[directoryObject](directoryobject.md) collection| The user that cloud joined the device or registered their personal device. The registered owner is set at the time of registration. Currently, there can be only one owner. Read-only. Nullable.|
 |registeredUsers|[directoryObject](directoryobject.md) collection| Collection of registered users of the device. For cloud joined devices and registered personal devices, registered users are set to the same value as registered owners at the time of registration. Read-only. Nullable.|
-|commands | [command](command.md) collection | Set of commands sent to this device|
-|usageRight|[usageRight](usageright.md) collection|Represents the usage rights a device has been granted. |
+|transitiveMemberOf |[directoryObject](directoryobject.md) collection| List the groups that the device is a member of. This operation is transitive. |
```
Comment (Lauragra) [inline api-reference/beta/resources/device.md:89]: The description still sounds like an operation and not a property.

File: api-reference/beta/resources/device.md [hunk 4]
```diff
@@ -81,11 +81,13 @@ This resource lets you add your own data to custom properties using [extensions]
 ## Relationships
 | Relationship | Type	|Description|
 |:---------------|:--------|:----------|
+|commands | [command](command.md) collection | Set of commands sent to this device|
```
Comment (Lauragra) [inline api-reference/beta/resources/device.md:84]: Add the missing period.

File: api-reference/beta/resources/device.md [hunk 5]
```diff
@@ -81,11 +81,13 @@ This resource lets you add your own data to custom properties using [extensions]
...
 |extensions|[extension](extension.md) collection|The collection of open extensions defined for the device. Read-only. Nullable.|
+|memberOf|[directoryObject](directoryobject.md) collection|Groups that this device is a member of. HTTP Methods: GET (supported for all groups). Read-only. Nullable.|
```
Comment (Lauragra) [inline api-reference/beta/resources/device.md:86]: This HTTP Methods part is somewhat non-standard. What do we mean here? Should this be a link to a GET topic that shows how to GET the groups the device is a member of?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**